lib/passport-wsfed-saml2/utils.js

```diff
@@ -4,6 +4,24 @@ var SamlAssertionParserError = require('./errors/SamlAssertionParserError');
 var SamlResponseParserError = require('./errors/SamlResponseParserError');
 var WSFederationResultParserError = require('./errors/WSFederationResultParserError');
 
+CERT_START = "-----BEGIN CERTIFICATE-----\n";
+const CERT_END = "\n-----END CERTIFICATE-----\n";
+const certToPEM = (cert) => CERT_START + cert.match(/.{1,64}/g).join('\n') + CERT_END;
+
+exports.certToPEM = certToPEM;
```

Comment:
To match style you can probably just inline this and the method definition above.